src/pages/reporting/ChargebackForm.tsx

```diff
@@ -94,6 +93,21 @@ export default () => {
   const dispatch = useDispatch();
   const chargeBackRequest = new ChargebackRequest();
 
+  const filteredGroupByOptions = () => {
```

Comment:
you can make optional fields in groupByOptions object. Then you won't need any logic here (https://blog.agney.dev/optional-spreading/)